rpcs3/Emu/Cell/SPUCommonRecompiler.cpp

```diff
@@ -543,12 +543,37 @@ extern void utilize_spu_data_segment(u32 vaddr, const void* ls_data_vaddr, u32 s
 		return;
 	}
 
-	for (u32 addr : obj.funcs)
+	if (spu_log.notice && !obj.funcs.empty())
 	{
-		spu_log.notice("Found SPU function at: 0x%05x", addr);
+		std::string to_log;
+		usz i = 0;
+
+		for (u32 addr : obj.funcs)
+		{
+			if (i == 0 && obj.size() < 8)
+			{
+				// Skip newline in this case
+			}
+			else if (i % 8 == 0)
+			{
+				to_log += '\n';
+			}
+			else
+			{
+				to_log += ", ";
+			}
+
+			fmt::append(to_log, "0x%08x", addr);
+			i++;
```

Comment:
lol. I realized this can just be a regular i++ for loop

Reply:
I mean yeah but whatever lol